solara/lab/components/chat.py

```diff
@@ -0,0 +1,198 @@
+import uuid
+from typing import Callable, Dict, List, Optional, Union
+
+from typing_extensions import Literal
+
+import solara
+from solara.components.input import use_change
+
+
+@solara.component
+def ChatBox(
+    children: List[solara.Element] = [],
+    style: Optional[Union[str, Dict[str, str]]] = None,
+    classes: List[str] = [],
+):
+    """
+    The ChatBox component is a container for ChatMessage components.
+    Its primary use is to ensure the proper ordering of messages,
+    using `flex-direction: column-reverse` together with `reversed(messages)`.
+
+    # Arguments
+
+    * `children`: A list of child components.
+    * `style`: CSS styles to apply to the component. Either a string or a dictionary.
+    * `classes`: A list of CSS classes to apply to the component.
+    """
+    style_flat = solara.util._flatten_style(style)
+
+    classes += ["chat-box"]
+    with solara.Column(
+        style={"flex-grow": "1", "flex-direction": "column-reverse", "overflow-y": "auto"},
+        classes=classes,
+        style_=style_flat,
```

Comment:
```suggestion
        classes=classes,
        style_=style_flat,
```

I thought this would make it more visible what the style is that we use.